tests/cases/compiler/inlineSourceMaps.ts

```diff
@@ -0,0 +1,6 @@
+// @target: ES3
+// @sourcemap: true
+// @inlinesourcemap: true
+
+var x = 0;
+console.log(x);
```

Comment:
I want to write unit tests for this function, cause other wise there is not much you can look at other than base-64 digits. will try to get it in later this week.